web/src/translation/i18n.ts

```diff
@@ -17,6 +17,14 @@ i18n
       order: ['querystring', 'localStorage', 'sessionStorage', 'navigator', 'htmlTag'],
       lookupQuerystring: 'lang',
       caches: ['localStorage'],
+      convertDetectedLanguage: (lng: string) => {
+        try {
+          return Intl.getCanonicalLocales(lng)[0];
```

Comment:
Hmm, we could go further here and strip out non letter characters as well (e.g. `DE).` -> `DE`) as this would help fix invalid markdown links and fix the concrete Sentry error you linked to :)

Reply:
We can try but we can't strip all non alphabetic letters, de-AT for example is a perfectly valid locale.

But I can whip up some simple validation.